.github/workflows/build.yml

```diff
@@ -13,12 +13,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - branch: swift-5.4.3-release
-            tag: 5.4.3-RELEASE
-
-          - branch: swift-5.5-release
-            tag: 5.5-RELEASE
-
```

Comment:
IMO this is unrelated to implementing `OpaquePointer`.

Reply:
This releases crash when attempting to build `OpaquePointer` code, as can be seen in #31. What would be the best way to make CI green on this PR without removing these releases from the build matrix?

Reply:
You can use `#if swift(>=5.6)`  around the implementation of `OpaquePointer` :)